tinyagentos/app.py

```
@@ -323,6 +323,29 @@ async def _probe_framework_versions() -> None:
+                try:
+                    deleted = await _store.sweep_expired()
+                    if _hub is not None:
+                        for mid, cid in deleted:
+                            await _hub.broadcast(cid, {
+                                "type": "message_delete",
+                                "seq": _hub.next_seq(),
+                                "channel_id": cid,
+                                "message_id": mid,
+                                "deleted_at": __import__("time").time(),
+                            })
+                except Exception as _e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)